packages/bridge-ui/src/constants/abi/TokenVault.ts

```diff
@@ -349,7 +349,7 @@ const abi = [
             type: "string",
           },
         ],
-        internalType: "struct TokenVault.CanonicalERC20",
+        internalType: "struct TokenVault.CannonicalERC20",
```

Comment:
i think the PR has been updated to say Canonical ? see https://github.com/taikoxyz/taiko-mono/pull/336